pkg/distro/fedora/imagetype.go

```diff
@@ -392,5 +392,10 @@ func (t *imageType) checkOptions(bp *blueprint.Blueprint, options distro.ImageOp
 		return nil, err
 	}
 
+	if customizations.GetFIPS() && !common.IsBuildHostFIPSEnabled() {
+		w := fmt.Sprintln(common.FIPSEnabledImageWarning)
+		return []string{w}, nil
```

Comment:
I think this also needs a log() like the RHEL versions. Unless there is a reason not to log on Fedora?

Reply:
There's no log in the whole function so I kept it the same way.

Reply:
I looked a bit deeper and I think no logging is ok for a warning. It's really meant to make it back to the user, who isn't going to read the logs. In most cases the warnings are currently ignored, but that can be fixed later. FWIW it does look like it makes it back up to the weldr API call and is returned as part of the compose start response.